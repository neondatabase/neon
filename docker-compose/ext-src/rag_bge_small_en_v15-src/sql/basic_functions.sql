-- Basic function tests
SELECT rag_bge_small_en_v15.chunks_by_token_count('the cat sat on the mat', 3, 2);
