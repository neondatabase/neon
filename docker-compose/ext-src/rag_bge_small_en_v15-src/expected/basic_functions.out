-- Basic function tests
SELECT rag_bge_small_en_v15.chunks_by_token_count('the cat sat on the mat', 3, 2);
                 chunks_by_token_count                  
--------------------------------------------------------
 {"the cat sat","cat sat on","sat on the","on the mat"}
(1 row)

